### PR TITLE
Change mingw-build to use download URLs

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

File: mingw-build.cmd [hunk 1]
```diff
@@ -9,21 +9,21 @@ set "PATH=C:\Program Files\7-Zip;C:\TDM-GCC-32-4.7.1\bin;%PATH%"
...
+set "CMAKE_DIRNAME=cmake-3.17.3-win32-x86"
+set "TINYXML2_URL=https://github.com/leethomason/tinyxml2/archive/7.1.0.tar.gz"
 set "TINYXML2_FILENAME=tinyxml2-7.1.0.tar.gz"
 set "TINYXML2_DIRNAME=tinyxml2-7.1.0"
-set "NSIS_URL=file://C:/Users/John/Downloads/nsis-3.04.zip"
-set "NSIS_FILENAME=nsis-3.04.zip"
-set "NSIS_DIRNAME=nsis-3.04"
+set "NSIS_URL=https://sourceforge.net/projects/nsis/files/NSIS%%203/3.05/nsis-3.05.zip/download"
+set "NSIS_FILENAME=nsis-3.05.zip"
+set "NSIS_DIRNAME=nsis-3.05"
```
Comment (jmeubank) [inline mingw-build.cmd:20]: I see that this version of NSIS deprecates ANSI targets. Until the NSIS scripts are ported to Unicode, let's use NSIS 3.04.

Comment (mimi89999) [reply mingw-build.cmd:20]: Did they change anything besides adding a warning? I was able to build a working installer without issues.

Comment (jmeubank) [reply mingw-build.cmd:20]: I prefer a "conservative" or "lazy" approach to versioning and dependencies here - only adjust the dependency if you can identify the specific change, feature, fix, or flaw it addresses, and do not adjust the dependency without addressing new unexpected behavior in the same pull request or branch. Thanks! 😃

Comment (mimi89999) [reply mingw-build.cmd]: Changed

**Pull request merged**